### ISSUE TITLE
x509 Certificate type mismatch causing build to fail

### ISSUE THREAD

**Issue by caleblloyd** [bloomberg/spire-tpm-plugin#5]
**Describe the bug**

x509 Certificate type mismatch causing build to fail in Go 1.14.2

**To Reproduce**

1. Use Go 1.14.2
2. Run `make`
3. Error:

```
cmd/server/tpm_attestor/tpm_attestor.go:170:39: 
cannot use leaf (type *"crypto/x509".Certificate) 
as type 
*"github.com/google/certificate-transparency-go/x509".Certificate 
in argument to 
"github.com/bloomberg/spire-tpm-plugin/pkg/common".GetPubHash
```

**Expected behavior**

Builds properly

**Screenshot**

![Screenshot from 2020-04-28 16-00-05](https://user-images.githubusercontent.com/2414837/80490089-82f6b880-8969-11ea-8d1d-14dbd233bb6d.png)

**Environment (please complete the following information):**
- Ubuntu 20.04
- Go 1.14.2
- `libtspi-dev` for trousers

**Comment by pawalt** [comment]
Hi @caleblloyd! Thanks for the catch. `go-attestation` uses multiple different x509 libraries, so I must have mixed them up in my last change. Once merged, #6 will fix this.

**Comment by pawalt** [comment]
Merged
